ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -62,14 +62,26 @@ jobs:
         timeout: 1800
         topology: *build
 
-  fedora-27/temp_commit:
+  fedora-27/temp_trust:
     requires: [fedora-27/build]
     priority: 50
     job:
       class: RunPytest
       args:
         build_url: '{fedora-27/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_trust.py
         template: *ci-master-f27
         timeout: 3600
-        topology: *master_1repl_1client
+        topology: *adroot_adchild_adtree_master_1client
+
+  fedora-27/temp_legacy_client:
```

Comment:
PRCI isn't able yet to run the legacy client tests because it doesn't have YAML files for a topology with the roles legacy_client_sssd_redhat role, legacy_client_nss_pam_ldapd_redhat or legacy_client_nss_ldap_redhat. You can remove this temp tests as it will not have any effect.